pkg/auth/migrate/rbac_to_acl.go

```diff
@@ -114,6 +118,35 @@ func RBACToACL(ctx context.Context, svc auth.Service, doUpdate bool, creationTim
 		messageFunc(group.DisplayName, *newACL, warnings)
 	}
 
+	users, _, err := svc.ListUsers(ctx, &model.PaginationParams{Amount: maxUsers + 1})
+	if err != nil {
+		return fmt.Errorf("list users: %w", err)
+	}
+	if len(users) > maxUsers {
```

Comment:
Not sure we can afford not to have pagination here!

Reply:
out of curiosity, why are we limiting the number of users/policies/groups/etc?

Reply:
changed to paginate over users and userPolicies, left groups as it was